mzLib/Test/FileReadingTests/TestMsDataFile.cs

```diff
@@ -126,22 +148,51 @@ public void DataFileTest()
             }
 
             int ok1 = 0;
-            foreach (var i in thefile.GetMsScansInTimeRange(0, 2))
+            foreach (var i in thefile.GetMsScansInTimeRange(0, 3))
                 ok1 += 1;
-
             Assert.AreEqual(1, ok1);
 
             int ok2 = 0;
             foreach (var i in thefile.GetMsScansInTimeRange(2, 4))
                 ok2 += 1;
-
             Assert.AreEqual(0, ok2);
 
             int ok3 = 0;
             foreach (var i in thefile.GetMsScansInTimeRange(-4, -2))
                 ok3 += 1;
-
             Assert.AreEqual(0, ok3);
+
+            MsDataScan theSpectrum2 = new MsDataScan(_mzSpectrumA, 1, 1, 
```

Comment:
please add comments in each case what is being tested and what value is being expected